soc/lowrisc/opentitan/rom_header.S

```diff
@@ -6,16 +6,23 @@
 
 #include <zephyr/toolchain.h>
 
-/* imports */
-GTEXT(__start)
+/* exports */
+GTEXT(__rom_header)
 
-/* OpenTitan manifest consists of 896 bytes (224 words) containing signature,
- * device ID, version info, etc. The test ROM ignores all of these fields
- * except for entry point (final word in manifest).
+/* OpenTitan manifest consists of 1024 bytes (256 words) of manifest
+ * containing signature, device ID, version info, etc. The test ROM
+ * ignores all of these fields except for entry point.
+ *
+ * The layout below conforms to manifest version: 0x71c3 (major),
+ * 0x6c47 (minor).
  */
 SECTION_FUNC(rom_header, __rom_header)
-	.rept(223)
+	.rept(225)
 	.word 0
 	.endr
 	/* Entry point is relative to the beginning of manifest. */
 	.word(__start - __rom_header)
+	/* Extensions entry table goes here. */
+	.rept(120)
```

Comment:
This should be .rept(30) (=120 bytes). 225 + 1 + 30 = 256 words * 4 bytes / word = 1024 bytes.

Reply:
oh gosh, math is hard.  thanks for the review.

Reply:
This could probably go into the comment, I'm sure it will be appreciated by the next person who tries to figure out what this value represents.

Reply:
+1 on this, let's add a comment as suggested by @ycsin. The next person to look at the code will thank us :)

Reply:
thanks for review - i added a reference to the manifest format as documented in the opentitan repository.

Reply:
Thanks!